README.md

```diff
@@ -222,4 +243,49 @@ Within OCP, you may use the OCP console to access the promql console to search f
 - ingressnodefirewall_node_packet_allow_total
 - ingressnodefirewall_node_packet_allow_bytes
 - ingressnodefirewall_node_packet_deny_total
-- ingressnodefirewall_node_packet_deny_bytes
\ No newline at end of file
+- ingressnodefirewall_node_packet_deny_bytes
+
+## Useful commands and tricks
+
+### Generating operator bundle
+
+In order to generate an operator bundle, run the following:
+```shell
+make bundle
```

Comment:
need to be followed by make manifests to cp bundle generated files to manifests/stable folder consumed by ART to build the images

Reply:
changed to:
```
make bundle
make manifests
```